src/FINDOLOGIC/Api/Responses/Xml21/Properties/Filter/Item/ColorItem.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties\Filter\Item;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use SimpleXMLElement;
+
+class ColorItem extends Item
+{
+    public function __construct(SimpleXMLElement $item)
+    {
+        parent::__construct($item);
+        $this->color = ResponseHelper::getStringProperty($item, 'color');
```

Comment:
The sub-classes themself should declare these properties, as they are only relevant for the specified item type.